<launch>
	<!-- example launch script for PX4 based FCU's with gcs_image_bridge -->

	<arg name="fcu_url" default="serial:///dev/ttyACM0:57600" />
	<arg name="gcs_url" default="udp://@" />
	<arg name="tgt_system" default="1" />
	<arg name="tgt_component" default="50" />

	<include file="$(find mavros)/launch/px4.launch">
		<arg name="fcu_url" value="$(arg fcu_url)" />
		<arg name="gcs_url" value="" />
		<arg name="tgt_system" value="$(arg tgt_system)" />
		<arg name="tgt_component" value="$(arg tgt_component)" />
	</include>

	<node pkg="mavros_extras" type="gcs_image_bridge" name="gcs_image_bridge" clear_params="true" output="screen">
		<!-- <remap from="/mavlink/gcs_image" to="/image/source" /> -->
		<param name="gcs_url" value="$(arg gcs_url)" />
	</node>
</launch>

<!-- vim: set noet fenc=utf-8 ff=unix ft=xml :
